src/tests.rs

```diff
@@ -138,7 +138,10 @@ release: {version}
 
 #[test]
 fn check_newlines() -> crate::Result<()> {
-    for file in walk_dir(get_cargo_workspace(), &[".git", "target"], |_| true) {
+    // tests contains the trycmd tests, which may have placeholders
+    for file in walk_dir(get_cargo_workspace(), &[".git", "target", "tests"], |_| {
+        true
+    }) {
```

Comment:
instead of skipping the whole `tests` dir, instead skip if the file extension is `.stderr` or `.stdout`